### ISSUE TITLE
deploy from examplesite didn't work

### ISSUE THREAD

**Issue by SYSUHPK** [mrhelloboy/seven#15]
The theme is really beautiful. I want to ask, following the deployment with the provided examples at https://hugoseven.netlify.app/, I don't know why the result is completely different from the provided examples, as shown in the figure below.
![微信截图_20241007222901](https://github.com/user-attachments/assets/7382336f-b2e7-43c4-92fd-0fa0f15ed0bc)

**Comment by SYSUHPK** [comment]
By deleting content, I can deploy successfully, but I don't understand why the following error is reported without deletion. 


```
ERROR deprecated: data.GetJSON was deprecated in Hugo v0.123.0 and will be removed in Hugo 0.136.0. use resources.Get or resources.GetRemote with transform.Unmarshal.
Built in 3974 ms
Error: error building site: render: failed to render pages: render of "section" failed: "D:\Workspace\Coding\seven\layouts\_default\goods.html:21:41": execute of template failed: template: _default/goods.html:21:41: executing "main" at <transform.Unmarshal>: error calling Unmarshal: no data to transform. 
```

I have made modifications in goods.html ，but the same error is still reported. What is the problem?
     
      
      {{ $goodsJson := resources.Get "goods/goods.json" }}
      {{ $goods := $goodsJson | transform.Unmarshal }}
      {{ range $goods.good }})

**Comment by mrhelloboy** [comment]
@SYSUHPK This bug has been fixed in v1.7.4.
